fic_modules/git.py

```diff
@@ -489,33 +505,38 @@ def filter_git_commit_keyword(repository_name, repository_path):
     last_checked = last_check(repository_name)
     new_commit_dict = {"0": {"lastChecked": str(datetime.utcnow())}}
     if limit_checker():
-        new_commits = GIT.get_repo(repository_path).get_commits(since=last_checked)
+        new_commits = GIT \
+            .get_repo(repository_path) \
+            .get_commits(since=last_checked)
     for commit in new_commits:
         files_changed_by_commit = [x.filename for x in commit.files]
         if files_changed_by_commit:
             each_commit = {}
             if LOGGER:
-                print(commit.commit.message)
+                LOGGER.info(commit.commit.message)
             if "deploy" in commit.commit.message:
                 number += 1
                 each_commit.update({int(number): get_commit_details(commit)})
                 new_commit_dict.update(each_commit)
     json_writer_git(repository_name, new_commit_dict)
-    return True
 
 
 def compare_versions(version_path, scriptworker_repo, latest_releases):
     """
-    checks the version of a scriptworker repo from its changelog, build-puppet and locally saved one against each other
+    checks the version of a scriptworker repo from its changelog, build-puppet
+    and locally saved one against each other
     :param version_path:
     :param scriptworker_repo:
     :param latest_releases:
     :return:
     """
 
-    version_in_puppet = get_version_from_build_puppet(version_path, scriptworker_repo)
+    version_in_puppet = \
```

Comment:
Really weird break here. I can't see any reason we start a variable with an escape (new line) character.
Please break after `get_version_from_build_puppet(version_path,` if line to long, break after `get_version_from_build_puppet(`
Example:
```python
version_in_puppet = get_version_from_build_puppet(version_path, 
                                                  scriptworker_repo)
```

Reply:
Fixed in the next commit.